src/api/app/models/update_notification_events.rb

```diff
@@ -14,20 +14,8 @@ def create_events
         e.elements('data') do |d|
           data[d['key']] = d['_content']
         end
-        retries = 10
-        begin
-          event = Event::Factory.new_from_type(type, data)
-          event.save!
-        rescue ActiveRecord::StatementInvalid => e
-          retries -= 1
-          if retries.positive?
-            Airbrake.notify("Failed to create Event : #{type.inspect}: #{data} #{e}")
-            retry
-          end
-          Airbrake.notify("Failed to create Event : #{type.inspect}: #{data} #{e}")
-        rescue StandardError => e
-          Airbrake.notify("Failed to create Event : #{type.inspect}: #{data} #{e}")
-        end
+        event = Event::Factory.new_from_type(type, data)
```

Comment:
How would you know *which* `Airbrake.notify` you look at?

Reply:
> How would you know _which_ `Airbrake.notify` you look at in errbit?

In errbit we could have received three different messages, starting with:
- 1: "Failed to create Event :"
- 2: "Failed to update PackageIssue :"
- 3: "Failed while running PackageUpdateIfDirtyJob:"

We only have received messages for this one: 2: "Failed to update PackageIssue :".

Now we are confident we potentially won't receive any "Failed to create Event :" messages, we don't need to identify _which_ `Airbrake.notify` we need to send. That's why we remove this unnecessary code to distinguish between `Aribrake.notify` options.

Reply:
Well that only means there was never *any* problem that required retrying right?

Reply:
There are cases of this still in errbit...

https://errbit.opensuse.org/apps/5620ca2bdc71fa00b1000000/problems/670d2a5584bf460c5bef1380
https://errbit.opensuse.org/apps/5620ca2bdc71fa00b1000000/problems/65f2e3fa84bf4608f8817fd6

Reply:
> Well that only means there was never _any_ problem that required retrying right?

Exactly. That's why don't need retrying.

Reply:
> There are cases of this still in errbit...
> 
> https://errbit.opensuse.org/apps/5620ca2bdc71fa00b1000000/problems/670d2a5584bf460c5bef1380 https://errbit.opensuse.org/apps/5620ca2bdc71fa00b1000000/problems/65f2e3fa84bf4608f8817fd6

Both cases happened due to general failures with the database, affecting the whole application. Nothing that retrying in this part of the code code make a difference.

Reply:
That may be but how do you know this never saved us? You can't because every exception looks the same...

Reply:
This could have saved us in the past: we had another version/set-up of the database and the codebase was different.

Now, just describing with other words the same written in the commit body, we don't need this code to handle in a different way potential failures happening accessing the database.

Fortunately, we have the backtrace for every exception saved in Errbit. Even if they look the same, we can always know were the exceptions have been triggered.